poker/hand.py

```diff
@@ -191,7 +191,12 @@ def is_pair(self):
 
     @property
     def shape(self):
-        return Shape(self._shape)
+        if self.is_pair:
+            return Shape.PAIR
+        elif self.is_suited:
+            return Shape.SUITED
+        else:
+            return Shape.OFFSUIT
 
```

Comment:
What was wrong with the original implementation? This seems much more complicated to me.

Reply:
With original implementation the following simple code fails:

$ cat test.py 
from poker.hand import Range, Hand

range = Range("AK")
hand = Hand('AKs')
if hand in range:
    print "In range"
else:
    print "Out of range"


$ python test.py
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    if hand in range:
  File "/usr/local/lib/python2.7/dist-packages/poker/hand.py", line 627, in __contains__
    return item in self._all_hands
  File "/usr/local/lib/python2.7/dist-packages/poker/hand.py", line 101, in __hash__
    return hash(self.first) + hash(self.second) + hash(self.shape)
  File "/usr/local/lib/python2.7/dist-packages/poker/hand.py", line 194, in shape
    return Shape(self._shape)
  File "/usr/local/lib/python2.7/dist-packages/poker/_common.py", line 28, in __call__
    return super(_PokerEnumMeta, cls).__call__(value)
  File "/usr/lib/python2.7/dist-packages/enum/__init__.py", line 347, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python2.7/dist-packages/enum/__init__.py", line 662, in __new__
    raise ValueError("%s is not a valid %s" % (value, cls.__name__))
ValueError: s is not a valid Shape

Reply:
What version of Python are you using?

Reply:
$ python --version
Python 2.7.15+

The same error happened here in travis 
https://travis-ci.org/pokerregion/poker/builds/572083053